src/constants/criteria/criteria-taste.js

```diff
@@ -0,0 +1,30 @@
+const expectation = {
+  title: 'Expectations',
+  description: 'Based on the essence of the dish what is delivered',
```

Comment:
Is it fine to add the price element to the 'essence of the dish'?

Reply:
Let's keep descriptions generic